hutch_python/cookiecutter/{{ cookiecutter.hutch }}/{{ cookiecutter.hutch }}env

```diff
@@ -1,17 +1,44 @@
 #!/bin/bash
 # Source this to load the full environment that hutch python uses
+# This file can be modified in several places for dev settings
 
-# edit this line only
-export CONDA_ENVNAME="pcds-3.0.1"
-export CONDA_BASE="/reg/g/pcds/pyps/conda/py36"
+# pcds-envs version
+export CONDA_ENVNAME="pcds-3.4.2"
+# In case you want dev dev
+#export CONDA_ENVNAME="dev"
+
+# Settings
 export HUTCH="{{ cookiecutter.hutch }}"
+export LOCAL_BASE="/u1/{{ cookiecutter.hutch }}opr/conda_envs"
+export NFS_BASE="/cds/group/pcds/pyps/conda/py36"
+USE_LOCAL="1"
 
+# Clear the user's old env variables
 unset PYTHONPATH
 unset LD_LIBRARY_PATH
 
-source "${CONDA_BASE}/etc/profile.d/conda.sh"
-conda activate "${CONDA_ENVNAME}"
+# Helpers
+LOCAL_ENV="${LOCAL_BASE}/${CONDA_ENVNAME}"
 HERE=`dirname $(readlink -f $0)`
+
+# Determine whether to use hard drive env or weka
+if [ -n "${USE_LOCAL}" ] && [ -d "${LOCAL_ENV}" ]; then
+  echo "Loading local disk python env ${CONDA_ENVNAME}"
+  source "${LOCAL_ENV}/bin/activate"
+else
+  echo "Loading WEKA python env ${CONDA_ENVNAME}"
+  source "${NFS_BASE}/etc/profile.d/conda.sh"
+  conda activate "${CONDA_ENVNAME}"
+fi
+
+# Standard dev path: local only
 export PYTHONPATH="${HERE}:${HERE}/dev/devpath"
+# Expanded hutch-python dev common path: changes specifically for hutch-python
+#export PYTHONPATH="${HERE}:${HERE}/dev/devpath:${HERE}/../common/dev/devpath"
+# Including dev_conda path: dev changes for other apps
+#export PYTHONPATH="${HERE}:${HERE}/dev/devpath:/cds/group/pcds/pyps/apps/dev/pythonpath/"
+
+# Activates pydaq, pycdb, pyami
 source pcdsdaq_lib_setup
```

Comment:
I've been having issues with this line manually sourcing env files with local installs. Can you give it a try please?

Reply:
Can you expand a bit? I'm not sure exactly what you mean.

Reply:
Sorry yeah that was kinda vague. Here's an example:
```
xpp-control:xpp pennebak$ source xppenv
Loading local disk python env pcds-3.4.2
bash: pcdsdaq_lib_setup: No such file or directory
(pcds-3.4.2) xpp-control:xpp pennebak$ ssh ioc-xpp-usr01
```

Reply:
thanks I can look into that

Reply:
This doesn't happen on my session at all- it just proceeds normally with no apparent error.

Reply:
Ok, something must just be wrong with my setup